### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by efb4f5ff-1298-471a-8973-3d47447115dc**
---
Added example of the closes keyword and a warning note
---

**Important note**
Please note that only PrestoN is able to merge Pull Requests into master.

**Pull Request Type**
Please select what type of pull request this is:
- [ ] Bugfix
- [x] Feature Implementation

**Related issue**
Please link the issue your pull request is referring to.
Closes https://github.com/FreeTubeApp/FreeTube/issues/1512
Related https://github.com/FreeTubeApp/FreeTube/issues/1541

**Description**
It added an example for linking issues in the related issue section.
It added a warning of incorrectly filling out the PR template or not filling out the PR template in the note section.

**Screenshots (if appropriate)**
Does not apply.

**Testing (for code that is not small enough to be easily understandable)**
Has this pull request been tested?
Please describe shortly how you tested it and whether there are any ramifications remaining. 
Yes.

**Desktop (please complete the following information):**
Does not apply.

**Additional context**
None.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -2,16 +2,17 @@
 Title
 ---
 
-**Important note**
+**Important notes**
 Please note that only PrestoN is able to merge Pull Requests into master.
+Please note that if u incorrectcly filled out or not filled out the PR template, it will result in the closure of this PR.
```
Comment (jasonhenriquez) [inline .github/PULL_REQUEST_TEMPLATE.md:7]:
```suggestion
We may remove your pull request if you do not use this provided PR template correctly.
```
Spelling errors, wording adjustment.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,16 +2,17 @@
 Title
 ---
 
-**Important note**
+**Important notes**
 Please note that only PrestoN is able to merge Pull Requests into master.
```
Comment (jasonhenriquez) [inline .github/PULL_REQUEST_TEMPLATE.md:6]:
```suggestion
**Important note**
```
We should probably remove this line at this point given that it is both inaccurate and does not provide salient information.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -2,16 +2,17 @@
...
+Please note that if u incorrectcly filled out or not filled out the PR template, it will result in the closure of this PR.
 
 **Pull Request Type**
 Please select what type of pull request this is:
 - [ ] Bugfix
 - [ ] Feature Implementation
 
 **Related issue**
-Please link the issue your pull request is referring to.
+Please link the issue your pull request is referring to. If this pull request fully resolves the relevant issue, put "closes" before the issue number. Example: "closes #99999". If this pull request fully resolves multiple relevant issues, put "closes" before both issue numbers. Example: "closes #99999, closes #12345".
```
Comment (jasonhenriquez) [inline .github/PULL_REQUEST_TEMPLATE.md:15]:
```suggestion
Please link the issue your pull request is referring to. If this pull request fully resolves the relevant issue, put "closes" before the issue number. Example: "closes #123456".
```

I think your second added sentence is not necessary given that this has only rarely been a problem.

**Review by jasonhenriquez — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: Yeah that makes sense. I guess i just wanted to be clear about the way it was done.

**Review by peepopoggers — APPROVED** [review]

**Review by jasonhenriquez — APPROVED** [review]

**Review by PikachuEXE — APPROVED** [review]

**Pull request merged**